Mathlib/GroupTheory/ArchimedeanDensely.lean

```diff
@@ -256,6 +256,16 @@ lemma denselyOrdered_units_iff {G₀ : Type*} [LinearOrderedCommGroupWithZero G
     refine ⟨Units.mk0 z hz'.ne', ?_⟩
     simp [← Units.val_lt_val, hz]
 
+/-- Any linearly ordered group with zero is isomorphic to adjoining `0` to the units of itself. -/
+@[simps!]
+def OrderMonoidIso.withZeroUnits {α : Type*} [LinearOrderedCommGroupWithZero α]
```

Comment:
Is commutativity needed?

Reply:
Yes, there is no linear ordered not-necessarily-comm group with zero.